### ISSUE TITLE
Warning when loading Media

### ISSUE THREAD

**Issue by Wizzel1** [aloisdeniel/media_gallery#18]
When Media get loaded into the GridView, I am getting this error : 

` E/HW-JPEG-DEC( 7108): [HME_JPEG_DEC_Delete](3321): HME_JPEG_DEC_Delete: decoder_ctx=null`

any Idea what that is?

**Comment by aloisdeniel** [comment]
Looks like a system event, but I never saw this.

Is the image working ? Does the application crash ?

**Comment by Wizzel1** [comment]
Yes, everything works as expected but I was wondering if this warning could lead to trouble.
